lib/rake/task_manager.rb

```diff
@@ -301,6 +321,33 @@ def get_description(task)
       desc
     end
 
+    # Return a value representing the "cost" of transforming str1 into str2
+    def levenshtein_distance(str1, str2)
```

Comment:
On the other hand, Rake seems to keep a safe distance from RubyGems (maybe for a reason).

This _could_ introduce some circular dependencies by depending on Gem itself.. and I'm not seeing anywhere besides `testtask` that uses it.

One could argue this method should be in a separate gem that both can require, and other people would find it useful. If that is the case, why not move it to core?